common/src/chain/transaction/account_outpoint.rs

```diff
@@ -160,3 +173,32 @@ impl AccountOutPoint {
         &self.account
     }
 }
+
+#[derive(
+    Debug,
+    Clone,
+    PartialEq,
+    Eq,
+    Ord,
+    PartialOrd,
+    Encode,
+    Decode,
+    serde::Serialize,
+    serde::Deserialize,
+)]
+pub enum OrderAccountCommand {
+    // Satisfy an order completely or partially.
+    // Second parameter is an amount provided to fill an order which corresponds to order's ask currency.
+    #[codec(index = 0)]
+    FillOrder(OrderId, Amount, Destination),
+    // Close an order and withdraw all remaining funds from both give and ask balances.
+    // Only the address specified as `conclude_key` can authorize this command.
+    // Amounts are important because they get into a signature so that such a tx cannot be applied
+    // to an altered state.
+    #[codec(index = 1)]
+    ConcludeOrder {
+        order_id: OrderId,
+        filled_amount: Amount,
+        remaining_give_amount: Amount,
```

Comment:
I will remove these fields is we go with `OrderData` in the signature

Reply:
Seems like I cannot do that because `OrderData` does not contain current balances

Reply:
removed